src/components/header.module.scss

```diff
@@ -0,0 +1,47 @@
+.header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #b48895;
+  height: 5rem;
+
+  &__input {
+    width: 70%;
+    height: 40%;
+    border-radius: 1rem;
+    background-color: #b48895;
+    border: 1px solid #000000;
+    transition: all 1s;
+    padding-left: 1rem;
+    margin-right: 1rem;
+  }
+  &__input:focus {
```

Comment:
You can use parent selector for :focus as well. For example
&__input {
&:focux {
}
}